staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/mutating_work_estimator.go

```diff
@@ -21,11 +21,18 @@ import (
 	"net/http"
 	"time"
 
+	"k8s.io/apiserver/pkg/apis/apiserver"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/util/flowcontrol/metrics"
 )
 
-func newMutatingWorkEstimator(countFn watchCountGetterFunc, config *WorkEstimatorConfig) WorkEstimatorFunc {
+const (
+	// TODO(wojtekt): Remove it once we tune the algorithm to not fail
+	// scalability tests.
```

Comment:
I thought @wojtek-t's intent is that the config file be able to disable this feature.  I think that is the right stance, as I have mentioned elsewhere.

Reply:
Removed.